svelte/src/lib/components/Application.svelte

```diff
@@ -213,3 +215,5 @@
 {/if}
 
 <AboutDialog bind:showDialog={showAboutDialog} />
+
+<DeleteTestCaseDialog bind:showDialog={showDeleteTestCaseDialog} />
```

Comment:
I dont think we need DeleteTestCaseDialog in Application.svelte  
(We already use it in FileExplorer/Node.svelte)